conda_build/skeletons/cpan.py

```diff
@@ -515,16 +578,16 @@ def deps_for_package(package, release_data, perl_version, output_dir,
         return build_deps, build_core_deps, run_deps, run_core_deps, packages_to_append
 
     for dep_dict in release_data['dependency']:
-        # Only care about requirements
+        # Only care about requirements outside the develop phase
         try:
-            if dep_dict['relationship'] == 'requires':
+            if dep_dict['relationship'] == 'requires' and dep_dict['phase'] not in ['develop']:
```

Comment:
As there is just one element there, you can just compare it
```suggestion
            if dep_dict['relationship'] == 'requires' and dep_dict['phase'] != 'develop':
```

Reply:
Yes.  I agree we should make this a simple compare to keep this PR separate from the next one where I add to the list and give new functionality.